### PR TITLE
Send custom commands to $SHELL

### PR DESCRIPTION
**Pull request by lausek**
This implements the feature requested in #158. If `$SHELL` was not set, we'll fall back to `sh`.

### CODE REVIEW

File: src/blocks/custom.rs [hunk 1]
```diff
@@ -115,7 +116,8 @@ impl Block for Custom {
         let mut update = false;
 
         if let Some(ref on_click) = self.on_click {
-            Command::new("sh").args(&["-c", on_click]).output().ok();
+            Command::new(env::var("SHELL").unwrap_or_else(|_| "sh".to_owned()))
```
Comment (RealOrangeOne) [inline src/blocks/custom.rs]: As it's intended to use a literal here, simply using `unwrap_or` here might be better?

Comment (lausek) [reply src/blocks/custom.rs]: Oh, yeah sure. Pretty new to this so I always appreciate comments ^^

**Comment by atheriel** [comment]
This implementation looks good to me, but I'm waiting on a response to my comments in the issue to before merging.

Also, if you have the time/energy, can you add a note to the `blocks.md` entry for this block about the new behaviour around `$SHELL`? (I won't block the merge on that request, however.)

**Comment by atheriel** [comment]
Looks good to me. Thanks! And my thanks to @RealOrangeOne for reviewing the initial code, too.

**Pull request merged**